requirements.txt

```
@@ -1 +1,2 @@
 dwave-ocean-sdk>=7.0.0
```

Might as well bump this up to 7.1.0.
```suggestion
dwave-ocean-sdk>=7.1.0
```